Assets/Scenes/SampleScene.unity

```diff
@@ -200,67 +200,399 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 519420028}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: 0, y: 0, z: -10}
-  m_LocalScale: {x: 1, y: 1, z: 1}
+  m_LocalPosition: {x: -808.71, y: 119.97, z: -10}
+  m_LocalScale: {x: 3, y: 3, z: 3}
   m_ConstrainProportionsScale: 0
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 0
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
---- !u!1001 &1227330104
+--- !u!1 &1433527202
```

Comment:
- [ ] You should add the Player prefab back to SampleScene and remove the Player GameObject you added to that scene after moving the health code to `PlayerController.cs`. Then, player movement and health will both be in SampleScene.